README.md

```diff
@@ -2,14 +2,37 @@
 
 This is an example of an [External Issuer] for cert-manager.
 
+## Install
+
+```
+kubectl apply -f https://github.com/cert-manager/sample-external-issuer/releases/download/v0.1.0/install.yaml
+```
```

Comment:
Once this merges, I'll do a `v0.1.0` release, and this instruction will be correct.